### PR TITLE
ES6 format

### PR DESCRIPTION
**Pull request by jjhesk**
update to babel ES6

Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

> **Unless you are a React Native release maintainer and cherry-picking an *existing* commit into a current release, ensure your pull request is targeting the `master` React Native branch.**

Explain the **motivation** for making this change. What existing problem does the pull request solve?

Prefer **small pull requests**. These are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Make sure tests pass on both Travis and Circle CI.

**Code formatting**

Look around. Match the style of the rest of the codebase. See also the simple [style guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#style-guide).

For more info, see the ["Pull Requests" section of our "Contributing" guidelines](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests).

### CODE REVIEW

File: Examples/UIExplorer/js/TextExample.ios.js [hunk 1]
```diff
@@ -23,27 +23,27 @@
...
-class Entity extends React.Component {
+
+class Entity extends Component {
   render() {
     return (
       <Text style={{fontWeight: '500', color: '#527fe4'}}>
         {this.props.children}
       </Text>
-    );
+    )
```
Comment (ide) [inline Examples/UIExplorer/js/TextExample.ios.js]: style: semicolon

File: Examples/UIExplorer/js/TextExample.ios.js [hunk 2]
```diff
@@ -23,27 +23,27 @@
...
-var {
+import React, {Component} from 'react';
+import ReactNative,{
   Image,
   StyleSheet,
   Text,
   View,
-  LayoutAnimation,
-} = ReactNative;
+  LayoutAnimation
```
Comment (ide) [inline Examples/UIExplorer/js/TextExample.ios.js]: style: trailing comma

File: Examples/UIExplorer/js/TextExample.ios.js [hunk 3]
```diff
@@ -23,27 +23,27 @@
 'use strict';
 
 const Platform = require('Platform');
-var React = require('react');
-var ReactNative = require('react-native');
-var {
+import React, {Component} from 'react';
+import ReactNative,{
```
Comment (ide) [inline Examples/UIExplorer/js/TextExample.ios.js]: remove the unused ReactNative import

File: Examples/UIExplorer/js/TextExample.ios.js [hunk 4]
```diff
@@ -23,27 +23,27 @@
 'use strict';
 
 const Platform = require('Platform');
```
Comment (janicduplessis) [inline Examples/UIExplorer/js/TextExample.ios.js]: You can import platform from react-native instead importing it with haste.

**Comment by hramos** [comment]
There's no test plan or description for why this is necessary. Some feedback was provided but after two months, no changes have been made. Closing as inactive/needs more info.

**Pull request closed without merging**